Dockerfile

```diff
@@ -639,7 +663,7 @@ COPY --from=sles12-build /google-cloud-ops-agent*.rpm /
 
 FROM opensuse/leap:15.1 AS sles15-build-base
 
-RUN set -x; zypper -n install git systemd autoconf automake flex libtool libcurl-devel libopenssl-devel libyajl-devel gcc gcc-c++ zlib-devel rpm-build expect cmake systemd-devel systemd-rpm-macros java-11-openjdk-devel unzip zip wget
+RUN set -x; zypper -n install git systemd autoconf automake flex libtool libcurl-devel libopenssl-devel libyajl-devel gcc gcc-c++ zlib-devel rpm-build expect cmake systemd-devel systemd-rpm-macros java-11-openjdk-devel unzip zip
 		# Add agent-vendor.repo to install >3.4 bison
 		RUN echo $'[google-cloud-monitoring-sles15-x86_64-test] \n\
```

Comment:
This would need to change as well, right (`x86_64`)? I'm pretty sure we can rename this to use `vendor` and ignore the arch, and use `$basearch` in the `baseurl` instead.

Reply:
(Or, since we won't be supporting ARM sles-15 in the first cut, can we just fail the build in this specific scenario? I don't have a sense of how big an effort it is to add ARM support for whatever this package is.)

Reply:
Decided to fail the build on ARM for sles-15 for now, we can add the repo later.

Reply:
You could have still used `$basearch` in the repo definition — the build would fall if the repo is nonexistent.

Reply:
Updated.